src/components/RecipesDisplay/RecipeBrief.css

```diff
@@ -0,0 +1,82 @@
+.recipe-brief-container{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  margin: 1.25em;  /* 20px */
+
+  font-family: Poppins;
+}
+
+.recipe-card{
+  width: 300px;
+  height: 225px;
+  margin-bottom: 1.94em; /* 31px */
+  border-radius: 24px;
+  background-position: center;
+  background:   linear-gradient(180deg, lightgray 50%, #000 73.77%), no-repeat center / cover;
+
+  transition: transform 0.5s ease, opacity 0.5s ease, box-shadow 0.5s ease; 
+
+  cursor: pointer;
+  margin-bottom: 1.94em; /* 31px */
+
+  /* shadow elevation 1 */
```

Comment:
it's better to define the variable instead of having it as a comment to what it is in your designs.

Reply:
removed comments.